Src/ReleaseHistory.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## *Unreleased*
```

Comment:
>Unreleased [](http://example.com/codeflow?start=4&length=10)

move this to 2.0.0-dev.

Reply:
be careful that it also might generate a merge conflict due to my previous change.

Reply:
Has 2.0.0-dev not been released yet?

Reply:
Not yet.